source/api/ut.pkb

```diff
@@ -64,6 +64,21 @@ create or replace package body ut is
   begin
     return ut_expectation_dsinterval(ut_data_value_dsinterval(a_actual), a_message);
   end;
+  
+  procedure fail(a_message in varchar2) is
```

Comment:
Fair point.
I'm wondering how the reporter will show the line of failure. I'm afraid it might be getting it wrong.